src/azure-cli/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -184,6 +184,16 @@ def validate_key_import_source(ns):
         raise ValueError('--pem-password must be used with --pem-file or --pem-string')
 
 
+def validate_key_import_type(ns):
+    # Default value of kty is: RSA
+    kty = getattr(ns, 'kty', None)
+    crv = getattr(ns, 'curve', None)
+
+    if (kty == 'EC' and crv is None) or (kty != 'EC' and crv):
```

Comment:
add a `lower()`?

Reply:
We don't accept the lower case, it also applies to other existing commands.